test/widgets/message_list_test.dart

```diff
@@ -439,7 +445,7 @@ void main() {
         ..statusCode = HttpStatus.ok
         ..content = kSolidBlueAvatar;
 
-      await setupMessageListPage(tester, messageCount: 10);
+      await setupMessageListPage(tester, messageCount: 1);
```

Comment:
One message instead of ten seems fine — but also I don't actually get any failure if I revert this change. So perhaps simplify the explanation by just leaving it out.